sdk/python/kfp/v2/compiler_cli_tests/test_data/lightweight_python_functions_v2_pipeline.json

```diff
@@ -245,5 +245,5 @@
     }
   },
   "schemaVersion": "2.1.0",
-  "sdkVersion": "kfp-1.8.9"
+  "sdkVersion": "kfp-1.8.11"
```

Comment:
nit: you don't need to update the v2 golden files, they are unrelated to this change. Leaving them alone can give you a smaller diff.